src/app/birthday/birthday.service.spec.ts

```diff
@@ -10,7 +10,38 @@ describe('Service: Birthday', () => {
     });
   });
 
-  it('should ...', inject([BirthdayService], (service: BirthdayService) => {
+  it('Service should exist', inject([BirthdayService], (service: BirthdayService) => {
     expect(service).toBeTruthy();
   }));
+
+  describe('getBirthdayFor behavior', () => {
+    it('Should return Date object', fakeAsync(() => {
+      let methodReturnedValue: Date;
+  
+      getService()
+        .getBirthdayFor('Dima')
+        .subscribe(event =>  methodReturnedValue = event);
+  
+      tick(5000);
+  
+      expect(checkIfDateReturned(methodReturnedValue)).toBeFalsy();
+    }));
+  
+    it('Should throw Error if name wasn`t provided or an empty string was provided', fakeAsync(() => {
+      const errorText: string = 'Unable to find birthday for empty name!';
+  
+      getService().getBirthdayFor('').subscribe(
+        date => fail(`Expected an error, not ${date}`),
+        error  => expect(error).toContain(errorText)
+      )
```

Comment:
Same here - simply define your spies callbacks and assert if one was called and other was not.